requirements.txt

```diff
@@ -50,7 +50,7 @@ packaging>=17.0
 
 # See issue #822 about issue in mock 4.0.3.
 mock>=3.0.0,<4.0.0; python_version < '3.6'
-mock>=3.0.0,<4.0.3; python_version >= '3.6'
+mock>=3.0.0; python_version >= '3.6'
```

Comment:
When combining the two commits, the merge conflict has been resolved. Please squash the two commits into one, it does not really make sense to have them separated.

Reply:
DONE